tango_simlib/compat.py

```diff
@@ -0,0 +1,36 @@
+# katcp.py
```

Comment:
Not sure what this line means? Is it supposed to be here - A bit misleading if you ask me.

Reply:
Removed comment that was copied over from katcp compat.py